### PR TITLE
Collect xen

### LINKED ISSUE #103: Collect xen
https://xenbits.xen.org/xsa/

### PR DESCRIPTION
**Pull request by Pushpit07**
Fixes #103 
Added code to import xen

Tried to make it run by modifying the `importer_yielder.py` and some other files but couldn't. The code is somewhat similar to the `ubuntu_usn` importer. Can't figure out why I'm getting an unexpected keyword argument

Screenshot 2021-05-27 at 19 29 54

### CODE REVIEW

File: vulnerabilities/fixtures/openssl.json [hunk 1]
```diff
@@ -40764,5 +40764,19 @@
                 ]
             }
         }
+    },
+    {
+        "model": "vulnerabilities.importer",
+        "pk": 17,
+        "fields": {
+            "name": "xen",
+            "license": "",
```
Comment (pombredanne) [inline vulnerabilities/fixtures/openssl.json:40773]: Do you know what the license would be?

**Review by pombredanne — COMMENTED** [review]
Thanks! this is looking great. One question though: this would only add new references to existing vulnerabilities and this based on the CVE?
Also we need to find out about the license for the data?

Also what could we do about `XSA-377` released in a few days/weeks on `2021-06-08 17:00` per https://xenbits.xen.org/xsa/ ? (I guess just waiting may be enough?)

File: vulnerabilities/fixtures/openssl.json (hunk 1 above)
Comment (Pushpit07) [reply vulnerabilities/fixtures/openssl.json:40773]: GPLv2

**Comment by Pushpit07** [comment]
> Also what could we do about `XSA-377` released in a few days/weeks on `2021-06-08 17:00` per https://xenbits.xen.org/xsa/ ? (I guess just waiting may be enough?)

Yes, we can wait

File: vulnerabilities/fixtures/openssl.json (hunk 1 above)
Comment (pombredanne) [reply vulnerabilities/fixtures/openssl.json:40773]: @Pushpit07

> re: GPLv2

if this is it, then state it. Do you have a link to the license info?

**Comment by pombredanne** [comment]
@Pushpit07 do think you want to finish this?
Other I can close for now and we can reopen later.

**Comment by Pushpit07** [comment]
> @Pushpit07 do think you want to finish this?
> Other I can close for now and we can reopen later.

Sorry for leaving in between. Will surely finish this. Give me a few days time

**Comment by pombredanne** [comment]
> Sorry for leaving in between. Will surely finish this. Give me a few days time

That's very nice of you! thank you ++

File: vulnerabilities/fixtures/openssl.json (hunk 1 above)
Comment (Pushpit07) [reply vulnerabilities/fixtures/openssl.json:40773]: https://wiki.xenproject.org/wiki/Xen_FAQ_General

**Comment by pombredanne** [comment]
@Pushpit07 Thanks! Think you missed  a signoff on a few commits... do you mind to amend these?

**Comment by pombredanne** [comment]
@Pushpit07 before we can merge, please just confirm here that your DCO signoff "Signed-off-by: Pushpit <pushpit07@gmail.com>" applies to all your commits. Thank you!

**Comment by Pushpit07** [comment]
Yes, it does

**Comment by pombredanne** [comment]
> Yes, it does

@Pushpit07 Thank you ++ , you rock :heart:

**Review by pombredanne — APPROVED** [review]
@Hritik14 I think we should merge and this can be completed afterwards.

**Comment by Hritik14** [comment]
@pombredanne Will do after fixing tests here

**Pull request merged**